CondensateCrew/condensate_backend
# Categories, Ideas, Actions, and IdeaCategories Tables

### What's this PR do?
- Added tables for categories, ideas, and actions, with a joins table for the ideas to categories many to many relationship

### Where should the reviewer start?
- All actual code changes are in `app/models`

### How should this be manually tested?
- To apply these changes locally, run `python3 manage.py db migrate` followed by `python3 manage.py db upgrade`. The changes should be viewable and relationships testable through Postico

Closes #14
